planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -91,6 +102,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.addApplicationInfo(adpYaml, slaInfoResponse, SLA_INFO_GROUPNAME);
 
         Map groups = (Map) adpYaml.remove(GROUPS);
+        adpYaml.remove(NODE_TYPES);
```

Comment:
I think @MicheleGuerriero is using the ADP NodeTypes [here](https://github.com/kiuby88/SeaCloudsPlatform/blob/feature/seaclouds-policy-generation/monitor/monitoring-dam-generator-core/src/main/java/eu/seaclouds/monitor/monitoringdamgenerator/MonitoringDamGenerator.java#L103-L104), so we should not delete NodeTypes from ADP. Of course, we should avoid NodeTypes in final DAM, but ADP needs its own  NodeTypes.